src/libmongoc/tests/json/crud/unified/aggregate-let.json

```diff
@@ -310,7 +207,8 @@
       "description": "Aggregate to collection with let option",
       "runOnRequirements": [
         {
-          "minServerVersion": "5.0"
+          "minServerVersion": "5.0",
+          "serverless": "forbid"
```

Comment:
As noted in the commit, this is a partial change for [CDRIVER-4063](https://jira.mongodb.org/browse/CDRIVER-4063) (from: [DRIVERS-1836](https://jira.mongodb.org/browse/DRIVERS-1836)).